import/src/test/java/com/google/cloud/healthcare/imaging/dicomadapter/CStoreServiceTest.java

```diff
@@ -392,7 +392,7 @@ public void testBackupUploadService_retryOn_DicomWebException408Code_Success() t
 
     verify(mockBackupUploader).doWriteBackup(any(InputStream.class), anyString());
     verify(mockBackupUploader, times(2)).doReadBackup(anyString());
-    verify(spyStowClient, times(2)).stowRs(any(InputStream.class));
+    verify(spyStowClient, atLeast(1)).stowRs(any(InputStream.class));
```

Comment:
Why would this ever happen only once?

Reply:
Yes, you right, my mistake. I thought that it`s the same case that in testBackupUploadService_ExceptionOnWriteToBackup where Exception thrown and stowRs call is processed in different threads. Reverted.